content/02.body.md

```diff
@@ -115,8 +115,8 @@ The edge prior is a numerical feature that can be computed for every pair of nod
 
 ### Analytical approximation of the edge prior
 
-We also considered whether the probability of an edge existing across permuted networks could be written as a closed form equation involving the node pair's degree.
-A major simplification is the assumption that the probability of an edge existing is independent of all other potential edges.
+We also considered whether the probability of an edge existing across permuted networks has a simple closed-form expression.
+To derive such an expression, we first make the simplifying assumption that the probability of any given edge existing is independent of all other potential edges.
```

Comment:
Should we be using past or present tense in this kind of writing? I know sometimes math seems to be present tense but sometimes past is preferred. Not sure when to prefer which.

Past would be:
```suggestion
To derive such an expression, we first made the simplifying assumption that the probability of any given edge existing is independent of all other potential edges.
```

Reply:
I'm glad you asked! The most important rule is to be consistent with whichever tense you pick, but I generally see publications written in the present tense. (A professor once told me to imagine you're narrating the proof to an audience as you write it on a board, "First we add this term here, then we make this assumption, then the expression is proved.", i.e., all present tense because the proof is unfolding before you, in the present.)

I had tried to keep my tense consistent with that of the rest of the section, which was past tense, but I much prefer to write things in present tense in papers, so I let this inconsistency slip through.

That said, we might be making some re-writes based on Blair's comments, so maybe we'll clean this up to be present tense.